modules/init.js

```diff
@@ -19,7 +19,7 @@ exports.start = function (mainWindow) {
     .then(()            => multiwallet.get())
```

Comment:
https://github.com/particl/partgui/blob/dev/modules/multiwallet.js#L26

Does this work on windows?

Reply:
ls -> linux?

Reply:
Hence the comment https://github.com/particl/partgui/blob/dev/modules/multiwallet.js#L22